src/translations/en.ts

```diff
@@ -7,6 +7,6 @@ export default {
     "refresh page":"refresh page",
     "Updated Book Details": "Updated Book Details",
     "Submit": "Submit",
-    
+    "Search..." : "Search..."
```

Comment:
wrong key, we should add "..." in key, keys shouldn't have any spaces also - please fix keys